cli/internal/context/testdata/duplicate-pkgs/package-lock.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "duplicate-names",
```

Align with the fixture name?
```suggestion
  "name": "duplicate-pkgs",
```